task_2/style.css

```
@@ -0,0 +1,60 @@
+.wrapper {
+    margin: 0 auto;
+    width: 700px;
```

To make it responsive we have to replace the fixed width with percentage 
Fluid layout 
example
```suggestion
    width: 60%;
```